### PR TITLE
add static condition

### PR DESCRIPTION
**Pull request by lemelon**
I've added some conditions that enable to prevent a static app to be started from a port.
So the steps are as followed : 
1. Clone the repository of a static app
2. Install the repository of a static app
And nothing more.

### CODE REVIEW

**Comment by frankrousseau** [comment]
@lemelon Please, can you write your PR description more properly?

File: server/lib/app.coffee [hunk 1]
```diff
@@ -1,6 +1,7 @@
 path = require "path"
 config = require('./conf').get
 fs = require 'fs'
+log = require('printit')()
```
Comment (frankrousseau) [inline server/lib/app.coffee]: When you use printit, you should at least mention a prefix.

**Comment by frankrousseau** [comment]
It looks ok. @poupotte can you look if it breaks anything?

File: server/controllers/applications.coffee [hunk 2]
```diff
@@ -75,7 +76,14 @@ module.exports.install = (req, res, next) ->
             log.error err.toString()
             sendError res, err, 400
         else
-            res.send 200, {"drone": {"port": result.port}}
+            # send path to home if it's a static app
+            if result.type is 'static'
+                res.send 200, {"drone": {
+                    "type": result.type
+                    "path": result.dir
+                }}
```
Comment (nono) [inline server/controllers/applications.coffee]: @lemelon what is this JS thing inside coffee?

```coffee
res.send 200, drone:
    type: result.type
    path: result.dir
```

does not work?

Comment (lemelon) [inline server/controllers/applications.coffee]: Oh sorry i apologize i will test it like this. Just started coffeescript, so still not very expert on the syntax

Comment (nono) [inline server/controllers/applications.coffee]: @lemelon don't worry. The review on PR are here to catch things like this and learn you good habits ;-)

**Comment by poupotte** [comment]
It's ok for me :)

File: tests/static_app_test.coffee [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+            it "When I try to install application", (done) ->
+                app =
+                    name: "front"
+                    repository:
+                        url: "https://github.com/lemelon/cozy-front.git"
+                        type: "git"
+                    type: "static"
+                client.post 'apps/front/install', app, (err, res, body) =>
+                    console.log '//////////////////////////'
```
Comment (aenario) [inline tests/static_app_test.coffee]: You should remove these debug logs

**Comment by frankrousseau** [comment]
@lemelon can you rebase your PR?

**Comment by poupotte** [comment]
Tests are break but it's unconnected with @lemelon changes. @frankrousseau, I'm ok to merge this PR, I'll fix tests ASAP.

**Pull request merged**